import_export/widgets.py

```diff
@@ -472,3 +475,76 @@ def clean(self, value, row=None, **kwargs):
     def render(self, value, obj=None):
         ids = [smart_str(getattr(obj, self.field)) for obj in value.all()]
         return self.separator.join(ids)
+
+class ResourceWidget(ManyToManyWidget):
```

Comment:
If we adopt that PR on post_save behavior it might remove the need to have this be a M2M inheriting field, and remove some of the annoying hack code.